src/Shared/test/SkipOnHelixAttribute.cs

```diff
@@ -16,10 +15,15 @@ public bool IsMet
         {
             get
             {
-                return !OnHelix();
+                // Skip 
+                var skip = OnHelix() && (Queues == null || Queues.Contains(GetTargetHelixQueue(), StringComparison.OrdinalIgnoreCase));
```

Comment:
This can get you odd partial matches. You should split on `;` and then do List.Contains.